supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -269,7 +269,9 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
       executionDir.toString.toFile.createIfNotExists(asDirectory = true, createParents = true)
 
       log.debug("{} Resolving job command", tag)
-      val command = localizeInputs(jobPaths.callRoot, runtimeAttributes.dockerImage.isDefined, fileSystems, jobDescriptor.inputs) flatMap {
+      val allowSoftLinks = configurationDescriptor.backendConfig.getBoolean("docker.allowSoftLinks")
+      val dockerLocEnabled = runtimeAttributes.dockerImage.isDefined && !allowSoftLinks
+      val command = localizeInputs(jobPaths.callRoot, dockerLocEnabled, fileSystems, jobDescriptor.inputs) flatMap {
```

Comment:
Yes, it will bring security concerns. That's why I'm discussing with some people in the CCC world why we should avoid use cases which require multiple file systems and big size files and they don't want these to be copied (that's why I added sl flag in dockerized jobs in the first place). I may remove this flag from the PR.